src/gui/processing/qgsprocessingtoolboxmodel.h

```diff
@@ -72,6 +75,7 @@ class GUI_EXPORT QgsProcessingToolboxModelNode : public QObject
       Group SIP_MONKEYPATCH_COMPAT_NAME( NodeGroup ), //!< Group node
       Algorithm SIP_MONKEYPATCH_COMPAT_NAME( NodeAlgorithm ), //!< Algorithm node
       Recent SIP_MONKEYPATCH_COMPAT_NAME( NodeRecent ), //!< Recent algorithms node
+      Favorite SIP_MONKEYPATCH_COMPAT_NAME( NodeFavorite ), //!< Favorites algorithms node, since QGIS 3.40
```

Comment:
```suggestion
      Favorite ), //!< Favorites algorithms node, since QGIS 3.40
```

We don't need compatibility code for a new value

Reply:
Fixed